extras/scripts/xslt-postwrite.md

```diff
@@ -0,0 +1,28 @@
+This post-write hook requires the Saxon HE processor. Follow the installation instructions, and place the file saxon9he.jar in your mik top-level folder. http://www.saxonica.com/html/documentation/about/gettingstarted/gettingstartedjava.html
+
```

Comment:
I had to think about this for a second. Can I suggest changing it to "place the file saxon9he.jar in your mik top-level MIK folder"?